{"name": "Попытка задания невалидного email", "status": "skipped", "statusDetails": {"message": "XFAIL reason: Известная проблема: API не валидирует email при обновлении профиля\n\n_pytest.outcomes.XFailed: Известная проблема: API не валидирует email при обновлении профиля", "trace": "self = <tests.test_user_profile.TestUserProfile object at 0x0000026968EE8160>, api_client = <helpers.api_client.StellarBurgersApi object at 0x0000026968FB2FF0>\nregistered_user = {'access_token': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY4MjAxMjU2OWVkMjgwMDAxYjVlMTlhNiIsImlhdCI6MTc0...m-pEY5DjlcR2vGaHbnw', 'email': 'user_a0ca0637@test.com', 'name': 'User_a0ca0637', 'password': 'Password_a0ca0637', ...}\ninvalid_email = 'space @email.com'\n\n    @allure.story(\"Ошибки валидации\")\n    @allure.title(\"Попытка задания невалидного email\")\n    @pytest.mark.parametrize(\"invalid_email\", [\n        \"invalid-email\",\n        \"missing@domain\",\n        \"space @email.com\"\n    ], ids=[\"no_at\", \"no_domain\", \"with_space\"])\n    def test_invalid_email_update(self, api_client, registered_user, invalid_email):\n        \"\"\"Проверка валидации email при обновлении\"\"\"\n        response = api_client.update_user_info(\n            access_token=registered_user[\"access_token\"],\n            email=invalid_email\n        )\n    \n        if response.status_code == HTTP_STATUS[\"OK\"]:\n            # Проверяем, что email действительно изменился (даже если невалидный)\n            assert response.json()[\"user\"][\"email\"] == invalid_email\n>           pytest.xfail(\"Известная проблема: API не валидирует email при обновлении профиля\")\nE           _pytest.outcomes.XFailed: Известная проблема: API не валидирует email при обновлении профиля\n\ntests\\test_user_profile.py:84: XFailed"}, "description": "Проверка валидации email при обновлении", "attachments": [{"name": "log", "source": "36d32f5d-3688-42df-9b07-8dfc2760d0ef-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "5fb1f65e-9bf0-4490-8ce4-e06f357b9aa0-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "invalid_email", "value": "'space @email.com'"}], "start": 1746932326809, "stop": 1746932326981, "uuid": "ece501fb-d1fc-4b12-bcda-885e1b9d825c", "historyId": "8f04c98b76e08193153633776ab5f2b1", "testCaseId": "f0a14ce03474488f7bcaefeb9e1b397d", "fullName": "tests.test_user_profile.TestUserProfile#test_invalid_email_update", "labels": [{"name": "story", "value": "Ошибки валидации"}, {"name": "feature", "value": "Изменение данных пользователя"}, {"name": "epic", "value": "Stellar Burgers API"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_profile"}, {"name": "subSuite", "value": "TestUserProfile"}, {"name": "host", "value": "Gemini_78"}, {"name": "thread", "value": "18408-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_profile"}]}